SpeakUp/ExtraGrammarUtility.cs

```diff
@@ -116,10 +116,10 @@ public static void ExtraRulesForPawn(string symbol, Pawn pawn, Pawn other = null
             }
 
             //childhood
-            MakeRule(symbol + "childhood", pawn.story.childhood.title);
+            MakeRule(symbol + "childhood", pawn.story.childhood.identifier);
```

Comment:
The identifier is replacing the childhood title. There would be only 1 Childhood variable. This change from pawn.story.childhood.title to pawn.story.childhood.identifier is because there are some childhoods that have different descriptions but the exact same title. pawn.story.childhood.identifier returns the title plus a unique number to reference a specific childhood.

Reply:
Oh, I see. My bad.